src/service/viewer-impl.js

```diff
@@ -790,15 +791,26 @@ export class Viewer {
   }
 
   /**
-   * Retrieves the Base CID from the viewer
+   * Get/set the Base CID from/to the viewer.
+   * @param {string=} opt_data Stringified JSON object {cid, time}.
    * @return {!Promise<string|undefined>}
    */
-  getBaseCid() {
+  baseCid(opt_data) {
```

Comment:
I think it's... probably fine. Did we have problem before? I imagine it's quite common to rename a method in core and the impl is in extension?